msteams-platform/sbs-teams-app-test-tool.yml

```
@@ -0,0 +1,351 @@
+### YamlMime:Tutorial
+title: Debug your bot using Teams App Test Tool
```

```suggestion
title: Debug your AI chat bot using Teams App Test Tool
```
This guide is not about debugging generic bot in test tool but specific to AI chat bot because it requires AI keys.
The description of the link to this page is also called `debug an AI chat bot using Teams App Test Tool.`

So I think it is better to change the title to `Debug your AI chat bot using Teams App Test Tool`

---

Updated as suggested